pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/creator/impl/SegmentColumnarIndexCreator.java

```diff
@@ -746,6 +748,9 @@ private void writeMetadata()
           } else {
             // No records in segment. Use current time as start/end
             long now = System.currentTimeMillis();
+            properties.setProperty(RAW_SEGMENT_START_TIME, String.valueOf(now));
```

Comment:
This is incorrect. We shouldn't set them here because millis value doesn't necessarily match the time format